src/routes/write-exercise/components/FeedbackSection.tsx

```diff
@@ -48,7 +49,9 @@ const Feedback = ({ result, document, submission }: FeedbackPropsType): ReactEle
   } else {
     Icon = IncorrectFeedbackIcon
     background = incorrect_background
-    message = `${labels.exercises.write.feedback.wrong} „${document.article.value} ${document.word}“`
+    message = needsToBeRepeated
+      ? labels.exercises.write.feedback.wrongForRetry
```

Comment:
What does `wrongForRetry` mean?

Reply:
If you enter the solution wrong for the first time you get the answer:
"That was wrong"
If you enter the solution wrong for the third time you get the answer:
"That was wrong, correct would be: der Hammer"

So if retrying you don't get the solution, you just can retry.

Reply:
Then I'd prefer some naming like `wrong` and `wrongWithSolution` or similar, that you are able to retry is not related to the label.

Reply:
changed it.